huggingface_dataset.ipynb

```diff
@@ -17,11 +17,14 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "nbsphinx": "hidden",
+    "tags": []
+   },
    "outputs": [],
    "source": [
     "!pip install -U pip\n",
-    "!pip install \"cleanvision[huggingface]\""
+    "!pip install \"cleanvision[huggingface] @ git+https://github.com/cleanlab/cleanvision.git\""
```

Comment:
I don't think we want users to be installing developer version of cleanlab for these?  I think we should try to keep this working for the latest stable release as much as we can.  Especially because we should be tagging releases of this examples repo to accompany the releases of the main cleanvision repo.

And if we ever update notebooks in a way that stable release version no longer works, then we can clarify in the main Readme file which notebooks require somebody to install latest developer version of cleanvision instead.